source/iterators.tex

```diff
@@ -3985,11 +3985,11 @@
         operator-(const move_iterator& x, const move_sentinel<S>& y);
     friend constexpr iter_rvalue_reference_t<Iterator>
       iter_move(const move_iterator& i)
-        noexcept(noexcept(ranges::iter_move(i.current)));
+        noexcept(noexcept(ranges::iter_move(i.base())));
     template<IndirectlySwappable<Iterator> Iterator2>
       friend constexpr void
         iter_swap(const move_iterator& x, const move_iterator<Iterator2>& y)
-          noexcept(noexcept(ranges::iter_swap(x.current, y.current)));
+          noexcept(noexcept(ranges::iter_swap(x.base(), y.base())));
```

Comment:
Ditto normative change equivalent to `noexcept(false)`.

Reply:
Good catch; reverted.